.github/workflows/build-cli.yml

```diff
@@ -60,12 +62,20 @@ jobs:
       - name: Build CLI
         env:
           CROSS_NO_WARNINGS: 0
+          RUST_LOG: debug
+          RUST_BACKTRACE: 1
+          CROSS_VERBOSE: 1
         run: |
           export TARGET="${{ matrix.architecture }}-${{ matrix.target-suffix }}"
           rustup target add "${TARGET}"
+          echo "Building for target: ${TARGET}"
+          echo "Rust toolchain info:"
+          rustup show
+          echo "Cross version:"
+          cross --version
```

Comment:
same question as above - probably wanna remove these?

Reply:
I don't think this is revealing anything bad but helps debug